tests/slack_sdk/web/test_web_client_logger.py

```diff
@@ -3,15 +3,16 @@
 
 from slack_sdk import WebClient
 from slack_sdk.web import base_client
+from tests.helpers import create_copy
+from tests.mock_web_api_server import cleanup_mock_web_api_server, setup_mock_web_api_server
 from tests.slack_sdk.web.mock_web_api_handler import MockHandler
-from tests.mock_web_api_server import setup_mock_web_api_server, cleanup_mock_web_api_server
 
 
 class TestWebClientLogger(unittest.TestCase):
     test_logger: logging.Logger
 
     def setUp(self):
-        self.test_logger = logging.Logger("test-logger")
+        self.test_logger = logging.getLogger("test-logger")
```

Comment:
> `WebClient` loggers must be initialized with [logging.getLogger](https://docs.python.org/3/library/logging.html#logging.getLogger) since all calls to this function with a given name return the same logger instance.

This is a great TIL!